iceoryx_hoofs/source/posix_wrapper/named_semaphore.cpp

```diff
@@ -40,11 +40,12 @@ static expected<void, SemaphoreError> unlink(const NamedSemaphore::Name_t& name)
         switch (result.error().errnum)
         {
         case EACCES:
-            IOX_LOG(ERROR) << "You don't have permission to remove the semaphore \"" << name << "\"";
+            IOX_LOG(ERROR, "You don't have permission to remove the semaphore \"" << name << '"');
             return err(SemaphoreError::PERMISSION_DENIED);
         default:
-            IOX_LOG(ERROR) << "This should never happen. An unknown error occurred while creating the semaphore \""
-                           << name << "\".";
+            IOX_LOG(ERROR,
+                    "This should never happen. An unknown error occurred while creating the semaphore \"" << name
+                                                                                                          << '"');
```

Comment:
You forgot the period :stuck_out_tongue_winking_eye:

Reply:
Will fix it in the follow up PR :)

Reply:
That's not necessary, IMO.

Reply:
Yeah, I also just noticed that there already were quite a few inconsistencies with the period